app/services/katello/repository_type_manager.rb

```diff
@@ -95,6 +146,15 @@ def find_content_type!(katello_label)
       def enabled?(repository_type)
         find(repository_type).present?
       end
```

Comment:
should this `enabled?` method be updated to mean 'usable'?    (and i guess maybe we could reconcile on one term. usable or enabled)

I believe the navigation configuration needs updating: https://github.com/Katello/katello/blob/master/lib/katello/plugin.rb#L94-L177

to have the check be done at runtime instead of startup time

Reply:
I think I like enabled better. I'll switch to that.

Reply:
> to have the check be done at runtime instead of startup time

I'm thinking of trying to update the menu every time I update the `enabled_repository_types`.  Does that sound like what you were thinking? I'm still looking into what that entails.

Reply:
that might work, the other option is that navigation menu items can accept a lambda https://github.com/theforeman/foreman/blob/develop/app/registries/menu/manager.rb#L60

and it could be looked up on demand.  We'd need to make sure its super fast though